Engine/AbortableRenderInfo.h

```diff
@@ -30,11 +30,6 @@
 
 #include "Global/GlobalDefines.h"
 
-#if !defined(Q_MOC_RUN) && !defined(SBK_RUN)
```

Comment:
are you certain shiboken 1 + shiboken 2 will continue working?
(we're still Qt4 until 2.6)

Reply:
Both versions still generate bindings as expected. So I'm certain about that.